x-pack/test/ui_capabilities/common/nav_links_builder.ts

```diff
@@ -15,6 +15,10 @@ export class NavLinksBuilder {
       management: {
         navLinkId: 'kibana:stack_management',
       },
+      // TODO: Temp until navLinkIds fix is merged in
```

Comment:
Is this still valid? Is there a link to the fix or something you could add here?

Reply:
It's still valid, ui_capabilities functional tests will fail without it. It's something I'd have to ask @legrego about in terms of timelines, but I think he has higher priority stuff going on and the refactor to fully move away from navLinkIds is fairly complicated from what I can tell (or at least for me 😅).